src/openrct2/core/Numerics.hpp

```diff
@@ -64,6 +65,27 @@ namespace OpenRCT2::Numerics
 #endif
     }
 
+    [[maybe_unused]] constexpr bool isPowerOf2(int v)
+    {
+        return v && ((v & (v - 1)) == 0);
+    }
+
+    // Rounds an integer down to the given power of 2. y must be a power of 2.
+    [[maybe_unused]] constexpr int floor2(const int x, const int y)
```

Comment:
What's up with the maybe_unused attribute? constexpr also means implicit inline

Reply:
Not too sure. I suppose it might have to do with the headers being compiled as a compilation unit on some of our CI jobs, to see if they carry all their dependencies. Happy to remove these attributes and see if that is indeed the case, though.

Reply:
If it were marked static it would definitely trigger "unused function" when compiled, yeah give it a try and we'll see.